docs/guides/mystira-shared-migration.md

```
@@ -0,0 +1,457 @@
+### Polly v8 Migration Issues
+
+**Error: `IAsyncPolicy` not found**
+- Update using statements to use `ResiliencePipeline<T>`
+- Replace `PolicyFactory` with `ResiliencePipelineFactory`
+
+**Error: `AddPolicyHandler` not found**
+- Replace with `AddStandardResilienceHandler()` or `AddResilienceHandler()`
+
+### Distributed Lock Timeouts
+
+**DistributedLockException: Could not acquire lock**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)